src/components/side-by-side-rendered-diff.jsx

```diff
@@ -39,6 +40,9 @@ export default class SideBySideRenderedDiff extends React.Component {
       ));
     }
 
+    transformA = compose(transformA, addTargetBlank);
+    transformB = compose(transformB, addTargetBlank);
```

Comment:
A slightly nicer improvement might be to make this part of `baseTransform` up on line 29, since we are always going to apply it:

```js
const baseTransform = compose(
  this.props.removeFormatting && removeStyleAndScript,
  addTargetBlank
);
```

That way you don’t have to repeat it twice.